.github/workflows/intergation-tests.yml

```diff
@@ -4,13 +4,16 @@ on:
     branches:
       - master
       - 'releases/*'
-  pull_request:
+  pull_request_target:
+    types: [labeled, opened]
     branches:
       - master
       - 'releases/*'
+      
 jobs: 
     trigger-integration-tests:
       name: Trigger Integration tests
+      if: startsWith(github.event.pull_request.head.label, format('{0}:',github.repository_owner)) || contains(github.event.pull_request.labels.*.name, 'safe to run test')
```

Comment:
Should we have a simpler tag `run test` or just `safe` maybe?

Reply:
run-test seems good